src/main/java/org/opensrp/web/rest/ClientResource.java

```diff
@@ -348,4 +348,33 @@ private int getTotal(ClientSearchBean clientSearchBean, AddressSearchBean addres
 		return new ResponseEntity<>(identifiers, HttpStatus.OK);
 	}
 	
+	/**
+	 * Fetch clients ordered by serverVersion ascending order
+	 *
+	 * @return a response with clients
+	 */
+	@RequestMapping(value = "/getAll", method = RequestMethod.GET, produces = {
+			MediaType.APPLICATION_JSON_VALUE })
+	public ResponseEntity<String> getAll(
+			@RequestParam(value = SERVER_VERSIOIN)  long serverVersion,
+			@RequestParam(required = false) Integer limit)
+			throws JsonProcessingException {
+
+		ClientSyncBean response = new ClientSyncBean();
+		List<Client> clients = clientService.findByServerVersion(serverVersion, limit);
+		response.setClients(clients);

```

Comment:
why are you returning via sync ClientSyncBean while only clients are being returned